src/LiveryBridge.ts

```diff
@@ -0,0 +1,296 @@
+/* eslint-disable @typescript-eslint/no-explicit-any --- MessageEvent returns an any object. The maps we use also use any, as multiple data types are allowed. */
+
+import { uuid } from './livery-interactive/util/uuid';
+
+export interface LiveryMessage {
+  id?: string;
```

Comment:
Why is this optional again?

Reply:
It was optional because some messages didn't need id's in earlier versions. Now this is no longer necessary.